### ISSUE TITLE
Registration button for new user only

### ISSUE THREAD

**Issue by dbauszus-glx** [GEOLYTIX/passport_glx#10]
We could check whether a user name exists and only display either a log-in or registration button based on the outcome of the check.

https://github.com/jaredhanson/passport-local/issues/4#issuecomment-274684239

**Comment by riclf** [comment]
Yes, /exist or similar url with a username, returning a return code. Don't want a graphic button or a hard code to a webpage as this POST or GET would be deep in coded logic, where the particular webpage would be invoked until after further processing.  It would feel to operate similar to /login but not require/use a password and not log the username in.  Just provide a return code.
